src/index.html

```diff
@@ -5,10 +5,62 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin = "anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="one">
+        <img
+          src = "images/moyo_1.svg"
+          alt = "logo"
+          class = "logo"
+        >
+      </a>
+
+      <nav>
+        <ul class="nav___list">
+          <li>
+            <a class= "nav___link is-active" href="two">Apple</a>
+          </li>
+
+          <li class="nav___item">
+            <a class= "nav___link" href="three">Samsung</a>
+          </li>
+
+          <li class="nav___item">
+            <a class= "nav___link" href="four">Smartphones</a>
+          </li>
+
+          <li class="nav___item">
+            <a class= "nav___link"
+              href="five"
+              data-qa="hover">Laptops & Computers</a>
```

Comment:
```suggestion
            <a
              class= "nav___link"
              href="five"
              data-qa="hover"
            >
              Laptops & Computers
            </a>
```
If you have three or more attributes then each of them should be in a new line and have two more space from the left side then openings tag, and closing bracket should have the same amount of spaces as opening tag